packages/teams-js/tsconfig.json

```diff
@@ -12,7 +12,8 @@
     "noImplicitReturns": true,
     "sourceMap": true,
     "declaration": true,
-    "strictNullChecks": true
+    "strictNullChecks": true,
+    "resolveJsonModule": true
```

Comment:
why was `resolveJsonModule` needed?

Reply:
`resolveJsonModule` is needed because we are importing `validDomains.json` into `constants` as a single source of truth for our domains. This way whenever we need to make changes or tests we only need to update the one file.